lib/util.js

```diff
@@ -10,19 +10,20 @@ Rye.define('Util', function(){
         }
 
     function each (obj, fn, context) {
+        var i = null
```

Comment:
This variable was declared twice in this function, should it just not have the `var` keyword on the second use?

I believed that just defining it prior to both would look better.